base/src/test/test_fn_or_xor.rs

```diff
@@ -0,0 +1,204 @@
+#![allow(clippy::unwrap_used)]
+
+use crate::test::util::new_empty_model;
+
+// These tessts are grouped because in many cases XOR and OR have similar behaviour.
+
+// Test specific to xor
+#[test]
+fn fn_xor() {
+    let mut model = new_empty_model();
+
+    model._set("A1", r#"=XOR(1, 1, 1, 0, 0)"#);
```

Comment:
It is ok to have those here, but you need to have them also in an xlsx file created by Excel. That way we are sure that is the desired behaviour.

You don't need the `r#"` that is used in rust when you have a `"` in the string and don't want to escape it.

Reply:
Will make those changes when I get a chance